scaladoc/src/dotty/tools/scaladoc/tasty/comments/markdown/SnippetRenderer.scala

```diff
@@ -0,0 +1,130 @@
+package dotty.tools.scaladoc.tasty.comments.markdown
+
+import com.vladsch.flexmark.html._
+
+import dotty.tools.scaladoc.snippets._
+
+case class SnippetLine(content: String, lineNo: Int, classes: Set[String] = Set.empty, messages: Seq[String] = Seq.empty):
+  def withClass(cls: String) = this.copy(classes = classes + cls)
+  private def escapeQuotes(msg: String): String = msg.flatMap {
+    case '"' => "&quot;"
+    case c => c.toString
+  }
```

Comment:
I wonder if it's not very slow. You could just use String.replace() or sth like this using StringBuilder:
https://stackoverflow.com/a/1234531

AFAIK we need to take care of run-time.

Reply:
aye aye